docs/static/releasenotes.asciidoc

```diff
@@ -0,0 +1,66 @@
+[[releasenotes]]
+== Release Notes
+
+This section summarizes the changes in the following releases:
+
+* <<logstash-5-3-0,Logstash 5.3.0>>
+
+[[logstash-5-3-0]]
+=== Logstash 5.3.0 Release Notes
+
+
+* Persistent queues:
+    ** Changed the default queue location on disk to include the pipeline's ID in the path hierarchy.
+       By default, the queue is now created under <path.data>/queue/main. This breaking change was made to
+       accommodate an upcoming feature where multiple, isolated pipelines could be run on the same Logstash
+       instance.
+    ** Added a _recovery_ process that runs when Logstash starts up to recover data written to the persistent
+       queue, but not yet checkpoint'd. This is useful in a situation when the input had written data to the queue,
+       but Logstash crashed before writing to the checkpoint file.
+    ** Added exclusive access to the persistent queue on disk, as defined by the path.queue setting. Using a file
+       lock we ensure that only a single Logstash instance has access to write to the queue on the same path, to
+       guard against corruption ({lsissue}6604[Issue 6604]).
+    ** You can now safely reload the pipeline config when using persistent queues. Previously, the reloading of
+       config could result in data corruption.  In 5.3, the reload sequence has been changed to reliably shutdown the
+       first pipeline before a new one is started with same settings.
+    ** Fixed an issue where Logstash would stop accepting new events when queue capacity is reached even though events
+       were successfully acknowledged ({lsissue}6626[Issue 6626]).
+
+* Fixed a warning message when --config.debug is used with --log.level=debug  ({lsissue}6256[Issue 6256]).
+
+
+[float]
+==== Input Plugins
+
+*`S3`*:
+
+* We now include the S3 key information in the metadata (https://github.com/logstash-plugins/logstash-input-s3/issues/105[Issue 105]).
+
+*`Unix`*:
+
+* Do not overwrite `host` and `path` fields if they are already provided by `add_field` config.
```

Comment:
Maybe this should say, "The `host` and `path` fields are no longer overwritten if they are already provided by `add_field` config."